### PR TITLE
biocaml.0.5.0 - via opam-publish

### PR DESCRIPTION
**Pull request by pveber**
The OCaml Bioinformatics Library
Biocaml aims to be a high-performance user-friendly library for
Bioinformatics. See the [website](http://biocaml.org) for detailed API
documentation. We welcome contributors and feedback from users. Please
contact us with any comments and suggestions for features you would
like added.
---
- Homepage: http://biocaml.org
- Source repo: https://github.com/biocaml/biocaml.git
- Bug tracker: https://github.com/biocaml/biocaml/issues
---

Pull-request generated by opam-publish v0.3.2

### CODE REVIEW

**Comment by dsheets** [comment]
Looks like you're now 4.03+ due to variant-inline-records:

```
### stderr ###
# /home/travis/.opam/system/lib/solvuu-build/solvuu.mk:3: _build/project.mk: No such file or directory
# File "lib/base/fasta.ml", line 80, characters 18-19:
# Error: Syntax error
# make: *** [_build/lib/biocaml_base.cma] Error 2
```

File: packages/biocaml/biocaml.0.5.0/opam [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  "ppx_sexp_conv"
+  "re"
+  "rresult"
+  "uri"
+]
+depopts: ["async" "core" "lwt" "ounit"]
+conflicts: [
+  "core" {< "111.13.00"}
+]
+available: [ocaml-version >= "4.01.0"]
```
Comment (yallop) [inline packages/biocaml/biocaml.0.5.0/opam]: @pveber, could you please update this to

```
available: [ocaml-version >= "4.03.0"]
```
?

**Comment by pveber** [comment]
Sorry, I missed the notifications from this PR. Thank you both for the reviews, I've updated the PR as you suggested.

**Comment by pveber** [comment]
Great, thanks!

**Pull request merged**